lisa/runner.py

```diff
@@ -94,11 +94,13 @@ class BaseRunner(BaseClassMixin, InitializableMixin):
 
     def __init__(
         self,
+        runbook_builder: RunbookBuilder,
         runbook: schema.Runbook,
```

Comment:
Remove the runbook, it can be generated from the runbook builder.

Reply:
I see. It needs to rename the `runbook` to `transformer_runbook`.

Reply:
Will that be confusing? We also use runbook in the transformer to mean `schema.Transformer`. By calling it transformer_runbook, it sounds to me like it is the `schema.Transformer` that gets passed into the Transformer.run(runbook), but that's not the case here.